### PR TITLE
Fix pointing info and NIRISS PSF wings

### PR DESCRIPTION
**Pull request by Johannes-Sahlmann**
This PR introduces a fix to the values of RA_V1 and DEC_V1 header values.

It also fixes a problem with the selection of NIRISS PSF wing files. This may have untested side-effects when simulating data with NIRCam. I verified that the code works for NIRISS and FGS.

### CODE REVIEW

File: mirage/psf/psf_selection.py [hunk 1]
```diff
@@ -287,7 +287,10 @@ def get_library_file(instrument, detector, filt, pupil, wfe, wfe_group,
 
             # Compare the header entries to the user input
             file_inst = header['INSTRUME'].upper()
-            file_det = header['DETECTOR'].upper()
+            try:
+                file_det = header['DETECTOR'].upper()
+            except KeyError:
+                file_det = header['DET_NAME'].upper()
```
Comment (mperrin) [inline mirage/psf/psf_selection.py]: Hmm, DET_NAME vs DETECTOR is probably our fault from writing inconsistent header keywords in different parts of WebbPSF. What keyword will DMS use for this? We can make sure WebbPSF is consistent from now on.

**Comment by Johannes-Sahlmann** [comment]
I checked by simulating NIRISS data that the pointing information is now correct. This ready for review.

File: mirage/psf/psf_selection.py (hunk 1 above)
Comment (bhilbert4) [reply mirage/psf/psf_selection.py]: DMS uses `DETECTOR`

**Comment by bhilbert4** [comment]
Closes #344

**Comment by Johannes-Sahlmann** [comment]
OK. I'll have a look at the failing tests.

**Comment by Johannes-Sahlmann** [comment]
Tests are passing. @bhilbert4 can you please merge?

**Pull request merged**